package.json

```
@@ -25,5 +25,5 @@
     "node": ">=7.6.0"
   },
   "repository": "https://github.com/bee-queue/arena.git",
-  "version": "2.6.2"
+  "version": "2.6.3"
```

version bumping happens on master, so we can release multiple changes together and comply with semantic versioning
```suggestion
  "version": "2.6.2"
```